2.describe-data/4.across-batch-comparisons.ipynb

```diff
@@ -1143,7 +1143,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 22,
```

Comment:
Again, you are fixing the old. Thank you! 

I believe this is what fueled some of our confusions before.

The previous state of the repo was incorrect!

Reply:
NVM! my bad